format/src/main/thrift/carbondata.thrift

```diff
@@ -45,6 +45,7 @@ struct BlockletBTreeIndex{
 struct BlockletMinMaxIndex{
     1: required list<binary> min_values; //Min value of all columns of one blocklet Bit-Packed
     2: required list<binary> max_values; //Max value of all columns of one blocklet Bit-Packed
+    3: optional list<byte> null_value;   // Null Bool turn on if null value present.
```

Comment:
Better use binary and try to use BitSet for null values

Reply:
we can use PresenceMeta to store null value

Reply:
Done. Stored as a Bitset